vms/avm/vm_test.go

```diff
@@ -577,13 +499,11 @@ func TestTxAcceptAfterParseTx(t *testing.T) {
 	require.NoError(parsedSecondTx.Verify(context.Background()))
 	require.NoError(parsedSecondTx.Accept(context.Background()))
 
-	firstTxStatus, err := env.vm.state.GetStatus(firstTx.ID())
+	_, err = env.vm.state.GetTx(firstTx.ID())
 	require.NoError(err)
-	require.Equal(choices.Accepted, firstTxStatus)
 
-	secondTxStatus, err := env.vm.state.GetStatus(secondTx.ID())
+	_, err = env.vm.state.GetTx(secondTx.ID())
 	require.NoError(err)
-	require.Equal(choices.Accepted, secondTxStatus)
```

Comment:
`GetTx` only returns `Accepted` transactions now.

Reply:
I removed the `DeleteStatus` from the `State` interface. Now - the `State` interface returns txs that were `AddTx`ed. The `State` interface no longer references statuses at all.